frontend/packages/console-app/src/components/nodes/menu-actions.tsx

```diff
@@ -17,9 +17,14 @@ export const MarkAsUnschedulable: KebabAction = (kind: K8sKind, obj: NodeKind) =
   },
 });
 
-export const MarkAsSchedulable: KebabAction = (kind: K8sKind, obj: NodeKind) => ({
+export const MarkAsSchedulable: KebabAction = (
+  kind: K8sKind,
+  obj: NodeKind,
+  resources: {},
+  { nodeMaintenance },
```

Comment:
how does the nodeMaintenance object/flag get here?

Reply:
`MarkAsSchedulable` action is used from metal3-plugin 'Bare Metal enhanced Node pages' [1]. That's where this is used from. Ideally I'd also apply the same behaviour on non bare metal Nodes but since Node Maintenance Operator is not part of OpenShift and the situation around it is not clear, I am keeping this behind the metal3-plugin for the time being.

[1] https://github.com/openshift/console/blob/master/frontend/packages/metal3-plugin/src/components/baremetal-nodes/menu-actions.ts#L75

Reply:
ok, can you please add a simple comment there as this is part of the nodes menu-actions?

Reply:
Done